care/facility/api/serializers/patient_consultation.py

```diff
@@ -94,6 +96,8 @@ class PatientConsultationSerializer(serializers.ModelSerializer):
         required=False, allow_null=True, allow_blank=True
     )
 
+    mews_field: MewsType = serializers.SerializerMethodField()
```

Comment:
Are you sure this is correct? Could you attach the screenshot of the swagger documentation showing the schema of this field in the PR body?

Reply:
yes this is correct, I attached in body

Reply:
I meant the schema, not the response output.

Reply:
how to check, please tell me the steps

Reply:
at endpoint /redoc ??

Reply:
no this is not showing there how to edit there, I'm not getting where that class SpectacularAPIView, SpectacularRedocView, SpectacularSwaggerView is, I can't find drf_spectacular.views

Reply:
Refer example: https://github.com/coronasafe/care/blob/develop/care/facility/api/serializers/asset.py#L129-L140

Reply:
@extend_schema_field(
    {
        "type": "object",
        "properties": {
            "hostname": {"type": "string"},
            "source": {"type": "string", "enum": ["asset", "location", "facility"]},
        },
        "nullable": True,
    }
)

so I just need to add annotations like this and no need to create other class like I did earlier in line mewsfield:mewstype

Reply:
![Screenshot (141)](https://github.com/coronasafe/care/assets/91010142/40f5df06-efdb-429c-9594-f346d73816bf)

done